tests/integration/records/single_db/test_records_load_datetime.py

```diff
@@ -0,0 +1,174 @@
+                self.assertEqual(date.year, SAMPLE_YEAR)
+                self.assertEqual(date.month, SAMPLE_MONTH)
+                self.assertEqual(date.day, SAMPLE_DAY)
+
+    def database_has_no_time_type(self) -> bool:
+        return self.engine.name == 'redshift'
+
+    def test_load_timeonly(self) -> None:
+        for timeformat in TIMEONLY_CASES:
+            self.load(hint_name='timeonlyformat',
+                      format_string=timeformat,
+                      column_name='time',
+                      field_type='time')
+            time = self.pull_result(column_name='time')
+            if self.database_has_no_time_type():
+                self.assertEqual(time, create_sample(timeformat))
+            elif self.database_provides_times_as_timedeltas():
+                assert isinstance(time, datetime.timedelta)
+                minutes = SAMPLE_HOUR*60 + SAMPLE_MINUTE
+                if 'SS' in timeformat:
+                    seconds = minutes*60 + SAMPLE_SECOND
+                else:
+                    seconds = minutes*60
+                self.assertEqual(time.seconds, seconds)
+
+            else:
+                self.assertEqual(time.hour, SAMPLE_HOUR)
+                self.assertEqual(time.minute, SAMPLE_MINUTE)
+                if 'SS' in timeformat:
+                    self.assertEqual(time.second, SAMPLE_SECOND)
+                else:
+                    self.assertEqual(time.second, 0)
+
+    def test_load_datetimetz(self) -> None:
+        for datetimetzformat in DATETIMETZ_CASES:
+            self.load(hint_name='datetimeformattz',
+                      format_string=datetimetzformat,
+                      column_name='datetimetz',
+                      field_type='datetimetz')
+            datetime = self.pull_result(column_name='datetimetz')
+            # if self.database_driver_provides_dates_and_datetimes_as_strings():
+            if isinstance(datetime, str):
+                # TODO
+                self.assertFalse(datetime)
+            self.assertEqual(datetime.year, SAMPLE_YEAR)
+            self.assertEqual(datetime.month, SAMPLE_MONTH)
+            self.assertEqual(datetime.day, SAMPLE_DAY)
+            self.assertEqual(datetime.hour, SAMPLE_HOUR)
+            self.assertEqual(datetime.minute, SAMPLE_MINUTE)
+            if 'SS' in datetimetzformat:
+                self.assertEqual(datetime.second, SAMPLE_SECOND)
+            else:
+                self.assertEqual(datetime.second, 0)
+
+    def test_load_datetime(self) -> None:
+        for datetimeformat in DATETIME_CASES:
+            self.load(hint_name='datetimeformat',
+                      format_string=datetimeformat,
+                      column_name='datetime',
+                      field_type='datetime')
+            datetime = self.pull_result(column_name='datetime')
+            # if self.database_driver_provides_dates_and_datetimes_as_strings():
+            if isinstance(datetime, str):
+                # TODO
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker